packages/components/src/styles/Group.module.css

```diff
@@ -0,0 +1,19 @@
+.group {
+  composes: input from './Input.module.css';
```

Comment:
[composes](https://lightningcss.dev/css-modules.html#class-composition) ftw. Going to refactor our focus states to utilize this.